### ISSUE TITLE
Wrong table name when generating insert...

### ISSUE THREAD

**Issue by robertmircea** [antonheryanto/dapper-rainbow-postgresql#1]
I'm having trouble with `Dapper.Rainbow.Postgresql`. I have the following relations:

``` c#
    class DB : Database<DB>
    {
        public Table<Batch> BatchStatistics { get; set; }
        public Table<Batch> Batches { get; set; } 
    }
```

When I run two inserts (one for each table), the second insert generates sql which refers to `Batches` table instead of `BatchStatistics`. TableName seems to be wrong (the same as the case of the first table).

db.Batches.Insert(new {...});
db.BatchStatistics.Insert(new {...});

This is a fragment from Postgresql server log. Notice the two insert statements which contain the same public.batch table name instead of the second one being public.batch_statistics:

``` sql
statement: INSERT INTO public.batch (name,...) VALUES (((E'Test')::text),...) RETURNING id
2015-03-23 22:09:12 EET LOG:  duration: 1.294 ms
2015-03-23 22:09:12 EET LOG:  statement: INSERT INTO public.batch (batch_id,pending) VALUES (((5)::int4),((0)::int4)) RETURNING id
2015-03-23 22:09:12 EET ERROR:  column "batch_id" of relation "batch" does not exist at character 27
```

**Comment by antonheryanto** [comment]
This is expected behavior, because Dapper.Rainbow map between Type => Table using Dictionary which Type is the key and table names is the value. if you would like to uses one type for multiple table considered inherited the types, example:

``` cs
public class BatchStat : Batch {}

public class : Database<DB>
{
    public Table<BatchStat> BatchStatistics { get; set; }
    public Table<Batch> Batches { get; set; } 
}
```

**Comment by robertmircea** [comment]
No...it's not about one type for multiple tables. I don't understand why you consider expected behaviour: I want two inserts into two separate tables. BatchStatistics and Batch are independent classes. The problem is that Dapper Rainbow generates the wrong sql statement for the second INSERT. It generates the sql with the name of the first table. When I inspect db.BatchStatistics.TableName property the value is `public.batch` instead of `public.batch_statistics`.
